<launch>

    <!-- <include file="$(find pointcloud_utils)/launch/shell_lidar_calibration.launch" /> -->

		<node pkg="pointcloud_utils" type="point_cloud_2_to_png.py" name="pc_grid_parser" required="true" output="screen" cwd="node">

      	   <param name="cloud_topic" 		value="/airsim_node/PhysXCar/lidar/VLP16_1" />
      	   <param name="target_frame" 		value="VLP16_1" />
      	   <param name="binary_mode" 		value="true" />
      	   <param name="publish_image" 		value="false" />
      	   <param name="save_image" 		value="true" />
      	   <param name="folder_name" value="filtered_clouds" />

      	   <param name="resolution" value="0.1" />
      	   <param name="map_height" value="1024" />
      	   <param name="map_width" value="1024" />

      	   <param name="x_max" value="100" />
      	   <param name="x_min" value="-20" />
      	   <param name="y_max" value="50" />
      	   <param name="y_min" value="-50" />
      	   <param name="z_max" value="100" />
      	   <param name="z_min" value="0"   />

      	  <!-- <param name="x_max" value="1000" />
      	   <param name="x_min" value="-20" />
      	   <param name="y_max" value="1000" />
      	   <param name="y_min" value="-51" />
      	   <param name="z_max" value="1000" />
      	   <param name="z_min" value="-1000"   /> -->
      	 </node>
</launch>